apps/test/util/netsimTestUtils.js

```diff
@@ -165,12 +165,16 @@ var fakeStorageTable = function () {
  * Fake set of storage tables for use in tests.
  */
 exports.fakeShard = function () {
+  var fakeChannel = {
+    subscribe: function () {}
```

Comment:
Likewise above; I'll add the unused parameters, and explicitly disable that JSHint option for these lines.